monarch-initiative/monarch-ui-new
# Text annotator and redesign icons

- replace custom tooltip directive with tippy (vue 3 version is still in alpha, which is probably why i didn't do this from the beginning, but it still seems to work fine so far)
- redesign/rework category icons in figma to make more consistent and only use stroke (no fill) and use `currentColor`
- upload latest backup of figma sketch file
- add options to icon component to add simple circular border around icon (for category icons)
- remove debounce from persist to avoid bug
- get rid of custom directive files (there should be no need for them besides a tooltip)
- add api call to text annotator biolink endpoint and parse/format results
- create upload button component that can do file uploads by browse or drag & drop
- css tweaks
- add util func to download json
- add text annotator HOC that shows results with tooltips, links, download button, etc, and has multiple upload methods including example text
- add tests
